openedx/core/djangoapps/content/__init__.py

```diff
@@ -2,3 +2,7 @@
 Setup the signals on startup.
 """
 import openedx.core.djangoapps.content.course_structures.signals
+
+# Importing this in this manner seems to resolve some inexplicable circular import issue.
+# pylint: disable=relative-import
+import course_structures
```

Comment:
That sounds ominous.  I can help diagnose those if you like.  Leaving this here gives me the heebie-jeebies.